extensions/ql-vscode/src/vscode-tests/no-workspace/query-history.test.ts

```diff
@@ -902,9 +948,9 @@ describe("query-history", () => {
               thisQuery,
               [thisQuery, allHistory[0]],
             );
-            assert(false, "Should have thrown");
+            fail("Should have thrown");
```

Comment:
Any idea why these tests aren't doing `rejects.toThrow` instead of try/catching?

Reply:
I've changed them to use `rejects` now, I think `fail` doesn't even properly work with `jest-circus`.